ReportPortal.Tests/ReportPortal.E2E.API.Tests/Scenarios/MsTest/LaunchesQueryByAdminMsTestTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using ReportPortal.E2E.API.Core.Business;
+using ReportPortal.E2E.API.Core.Business.Models.Responses;
+using ReportPortal.E2E.Common.Extensions;
+
+namespace ReportPortal.E2E.API.Tests.Scenarios.MsTest
+{
+    [TestClass]
+    public class LaunchesQueryByAdminMsTestTests : MsTestBaseTest
```

Comment:
Design and implement test data storing utility and place it in Core Layer.

Reply:
Fix for all test data and all tests

Reply:
Done. Sorry, didn't catch that it should be for all tests